### ISSUE TITLE
can't build on January 2024 Slackware

### ISSUE THREAD

**Issue by yurytch** [Ray-V/tde-slackbuilds#35]
Maybe it's just me, but first time in several years I can't build the requirements part of TDE, starting with TQT3. My system is January 2024 Slackware, with multilib. I'm trying to build 14.1.2 release

[tqt3-14.1.2-x86_64-1-build-log.gz](https://github.com/Ray-V/tde-slackbuilds/files/15167674/tqt3-14.1.2-x86_64-1-build-log.gz)

**Comment by Ray-V** [comment]
For the 14.0.13 release I added QMAKE_INCDIR to the qmake.conf patch in tqt3.SlackBuild.  
Can't remember now why, but I think it was because of the different environment variables TQTDIR for development builds and QTDIR for release builds causing build problems and to match -headerdir as set in configure.

That difference is no longer the case and I can reproduce a lot of the errors with R14.1.1 installed because the inclusion of QMAKE_INCDIR in qmake.conf is picking up headers from that previous installation which are not compatible with R14.1.2.
  
So, option 1 would be to remove any previous installation of TDE.  
This is what I did when testing the 14.1.2 build and everything built ok.  

Option 2 - with a previous TDE installed  
Remove these lines from the tqt3.SlackBuild qmake.conf patch
```
@@ -40 +40 @@
-QMAKE_INCDIR		=
+QMAKE_INCDIR		= $(TQTDIR)/include/tqt
```
Will build tqt3 and tqtinterface ok, but untested beyond that and other packages will need updating for compatibility with the new tqt headers.

I don't have an up-to-date Slackware installed, so if there are still build errors, we'll need to look at possible problems building with a later Slackware.
